PhysicsTools/NanoAOD/interface/SimpleFlatTableProducer.h

```diff
@@ -325,6 +326,57 @@ class SimpleFlatTableProducer : public SimpleFlatTableProducerBase<T, edm::View<
   std::vector<std::unique_ptr<ExtVariable<T>>> extvars_;
 };
 
+template <typename T>
+class BXVectorSimpleFlatTableProducer : public SimpleFlatTableProducerBase<T, BXVector<T>> {
```

Comment:
please add a fillDescription to this new module,

Reply:
of course, I didnt do it as it hadnt been done for the others but thats because I was basing of 12_4_X, I see its all done now in the main branch so I can easily make that change.